forest/src/cli/config.rs

```diff
@@ -31,12 +29,17 @@ pub struct Config {
     pub encrypt_keystore: bool,
     /// Metrics bind, e.g. 127.0.0.1:6116
     pub metrics_address: SocketAddr,
+    /// RPC bind, e.g. 127.0.0.1:1234
+    pub rpc_address: SocketAddr,
     pub rocks_db: forest_db::rocks_config::RocksDbConfig,
     pub network: Libp2pConfig,
     pub sync: SyncConfig,
     pub chain: Arc<ChainConfig>,
 }
 
+// 127.0.0.1
+const LOCALHOST: IpAddr = IpAddr::V4(Ipv4Addr::new(127, 0, 0, 1));
```

Comment:
Do we need it? There's a constant `std::net::Ipv4Addr::LOCALHOST` already there for grabbing.

Reply:
I learn something new every day. :)